UIforETW/UIforETWDlg.cpp

```diff
@@ -1131,7 +1940,7 @@ void CUIforETWDlg::OnSize(UINT nType, int /*cx*/, int /*cy*/)
 		int yDelta = windowRect.Height() - lastHeight_;
 		lastHeight_ += yDelta;
 
-		UINT flags = SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE;
+		const UINT flags = (SWP_NOMOVE | SWP_NOOWNERZORDER | SWP_NOZORDER | SWP_NOACTIVATE);

```

Comment:
The parentheses don't seem to add any clarity (to my eye) in this case. My recommendation for borderline style changes would be to not make them, because their value is probably less than the cost of the churn. On the other hand, if you wrote a new line of code with the parentheses I wouldn't change it, for the same churn-avoidance reasons.

That is, if a change has a clear benefit (whatever *that* means) then make it. If it is style issue where reasonable people would disagree then avoiding churn wins.

I don't know if that explanation is any use since it depends completely on a subject understanding of clarity, value, style, etc., but I thought I'd share my thinking.